packages/node_modules/@webex/plugin-rooms/src/rooms.js

```diff
@@ -241,6 +241,84 @@ const Rooms = SparkPlugin.extend({
       .then((res) => new Page(res, this.spark));
   },
 
+  /**
+   * Returns a list of rooms with details about the data of the last
+   * actvity in the space, and the date of the users last presences in
+   * the room.
+   *
+   * For spaces where lastActivityDate > lastSeenDate the space
+   * can be considerd to be "unread"
+   *
+   * This differs from the rooms.list() function in the following ways:
+   *  -- when called with no parameters it returns an array of all
+   *     spaces that the user is a member of
+   *  -- no other options, eg: max, are considered
+   *  -- pagination is not supported. ALL rooms are returned which
+   *     can result in a large payload
+   *  -- only "id", "lastActivityDate", and "lastSeenDate" are
+   *     guaranteed to return
+   *  -- "title" is returned for group spaces only to improve
+   *     efficiency.  Clients that detect unread status for spaces
+   *     with no displayName can query the space details by id
+   *
+   * In general this function should be used only when the
+   * client needs to access read status info, for example on
+   * startup.   After startup track message and membership:seen
+   * events to maintain read status client side.
+   *
+   * This function may be deprecated when this info is provided in the membership
+   * objects returned in the list function.
+   * @instance
+   * @memberof Rooms
+   * @returns {Promise<RoomObjectList>}
+   */
+  listWithReadStatus() {
+    return this.spark.internal.conversation.list({},
+      {
+        activitiesLimit: 0,
+        participantsLimit: 0,
+        isActive: true,
+        deferDecrypt: true
```

Comment:
with deferDecrypt don't you need to decrypt the payload somewhere?

Reply:
This is admitedly hacking on my part.   I "discovered" that if I set `deferDecrypt=true` that the response has the room title unencrypted in the `displayName` field for each conversation (that is not a one on one space).